docs/feature_ps2_mouse.md

```diff
@@ -88,6 +88,32 @@ In your keyboard config.h:
 #endif
 ```
 
+### Interrupt Version (ARM chibios)
+
+pretty much any two pins can be used for the (software) interrupt variant on ARM cores, the example below uses A8 for clock, and A9 for data.
+
+In rules.mk:
+
+```
+PS2_MOUSE_ENABLE = yes
+PS2_USE_INT = yes
+```
+
+In your keyboard config.h:
+
+```
+#define PS2_CLOCK A8
+#define PS2_DATA  A9
+#define PS2_CLOCK_PORT EXT_MODE_GPIOA
+```
+the last define is used to select wich port DATA and CLOCK lines are wired to, and therefor the software-pin-interrupt should be configured for (replace the 'A' with your port-of-wiring-choice)
+
+And in the chibios specifig halconf.h:
+```
```

Comment:
```suggestion
```h
```

Reply:
Why are we adding h in the end?

Reply:
It adds colour to the code for readability. Compare without:

```
#define HAL_USE_EXT                 TRUE
```

And with:

```h
#define HAL_USE_EXT                 TRUE
```

Reply:
Actually this should use 'c' instead of 'h', see <https://github.com/github/linguist/blob/master/lib/linguist/languages.yml> for the list and alias'.

Reply:
Ah yes, I mixed up alias and extension.

Reply:
done - thanks for the hint :-)